components/AlertNotification/AlertNotification.test.tsx

```diff
@@ -30,7 +30,12 @@ const variantTestCases = {
 describe('AlertNotification', () => {
   for (const [key, value] of Object.entries(variantTestCases)) {
     it(`renders the correct variant ${key}`, () => {
-      render(<Alert variant={AlertVariants[key]} message={value.message} />);
+      render(
+        <Alert
+          variant={AlertVariants[key as keyof typeof AlertVariants]}
```

Comment:
Was this test failing because of type errors?

Reply:
No, it was just a TypeScript error in my IDE.